AndelaOSP/wire
# Ft countries filter

#### What does this PR do?
This PR adds functionality to the countries filter button.
#### Description of Task to be completed?
Create an incident sort according to countries
#### How should this be manually tested?
Run the project on your local machine
On the dashboard, click on the countries button.
Select a country of choice to see incident reports for that particular location.
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#154080529
